tests/test_startup.py

```diff
@@ -69,12 +69,13 @@ def test_on_end(tiny_text_config):
     task_manager.clear_worker_cache.assert_called_once()
 
 
-def test_startup_task_no_train(tiny_text_config_no_train, tiny_text_task_manager):
-    dms = load_dataset_split_managers_from_config(tiny_text_config_no_train)
+def test_startup_task_one_ds(tiny_text_config_one_ds, tiny_text_task_manager):
+    dms = load_dataset_split_managers_from_config(tiny_text_config_one_ds)
     assert DatasetSplitName.eval in dms and DatasetSplitName.train in dms
 
     mods = startup_tasks(dms, tiny_text_task_manager)
-    assert all("train" not in k or "eval" in k for k in mods.keys())
+    ds_name, other_ds_name = get_tiny_text_config_one_ds_name(tiny_text_config_one_ds)
+    assert all(other_ds_name not in k or ds_name in k for k in mods.keys())
```

Comment:
I'm not sure I understand this assertion. Shouldn't it be `and` instead of `or`?
```suggestion
    assert all(other_ds_name not in k and ds_name in k for k in mods.keys())
```

Reply:
It's because some keys have neither because they are computed on `DatasetSplitName.all`. Let me make that explicit. LMK what you think.

Reply:
Ah-ha! I see. As it was, the ` or ds_name in k` had no effect since `other_ds_name not in k` was always true. Thanks for the explanation.

Reply:
In other words, if the key was `potato1`, `potato2`, etc., the test would have still passed.